test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -1494,20 +1494,18 @@ public void PushCommand_PushToServerV3_ApiKeyFromConfig()
                                                     <configuration>
                                                       <packageSources>
                                                         <add key='nuget.org' value='{0}' protocolVersion='3' />
-                                                        <add key='nuget.org' value='{1}' />
                                                       </packageSources>
                                                       <packageRestore>
                                                         <add key='enabled' value='True' />
                                                         <add key='automatic' value='True' />
                                                       </packageRestore>
                                                       <disabledPackageSources />
                                                       <apikeys>
-                                                        <add key='{2}' value='{3}' />
+                                                        <add key='{1}' value='{2}' />
```

Comment:
Can we validate that we can still push to
1. NuGet.org V2
2. NuGet.org V3

when both feeds are in nuget.config and when only one of them is in?